community-members.md

```diff
@@ -150,13 +150,18 @@ Repo: [open-telemetry/opentelemetry-php](https://github.com/open-telemetry/opent
 
 The list of active members (both "approvers" and "maintainers") for the OpenTelemetry PHP API and SDK can be found in the [open-telemetry/opentelemetry-php CONTRIBUTING file](https://github.com/open-telemetry/opentelemetry-php/blob/master/CONTRIBUTING.md).
 
+## Swift
+
+Repo: [open-telemetry/opentelemetry-swift](https://github.com/open-telemetry/opentelemetry-swift)
+The list of active members (both "approvers" and "maintainers") for the OpenTelemetry Swift API & SDK can be found in the [open-telemetry/opentelemetry-swift CODEOWNERS file](https://github.com/open-telemetry/opentelemetry-swift/blob/main/CODEOWNERS).
```

Comment:
I'd suggest documenting in README.md or CONTRIBUTING.md similar to other repos

Reply:
I'll update this to link to the readme and update the readme later